<?xml version="1.0"?>
<!-- Mesh file and approximate numbers are from Abe Bachrach at Skdio.  -->
<robot name="quadrotor">
  <!--
  Axes are standard vehicle coordinates:
    +X - Pointing toward the nose (primary camera).
    +Y - Towards the left rotors.
    +Z - Towards the top of the vehicle..
  Origin:
    (0, 0, 0) in the center of the main quadrotor body.
  -->
  <link name="base_link">
    <inertial>
      <mass value="0.775"/>
      <origin xyz="0 0 0"/>
      <inertia ixx="0.0015" ixy="0.0" ixz="0.0" iyy="0.0025" iyz="0.0" izz="0.0035"/>
    </inertial>
    <visual>
      <origin rpy="1.570796 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://drake_models/skydio_2/skydio_2_1000_poly.obj" scale=".00254"/>
      </geometry>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size=".36 .4 .06"/>
      </geometry>
    </collision>
  </link>
</robot>
